src/main/resources/site/processors/add-metadata.js

```diff
@@ -9,134 +9,137 @@ const view = resolve("add-metadata.html");
 
 exports.responseProcessor = function (req, res) {
     const site = libs.portal.getSite();
+    const siteConfig = libs.portal.getSiteConfig();
     const content = libs.portal.getContent();
-    const siteConfig = libs.common.getTheConfig(site);
 
-    const isFrontpage = site._path === content._path;
-    const pageTitle = libs.common.getPageTitle(content, site);
-    const titleAppendix = libs.common.getAppendix(site, isFrontpage);
+    /** We don't want 500 on controllers without content */
+    if (content) {
```

Comment:
```
if (!content) {
return ...
}
```

and you won't need to right shift so much code

Reply:
I don't like the pattern of having return statments all over the code body. But just at the beginning and end is fine.

Reply:
I'm with @rymsha on this one. If we want single return then the method should have been refactored so that everything inside `if (content) {...}` is a separate function, otherwise it becomes spaghetti.

Reply:
Did you read everything i wrote? I'm fine with this being at the beginning and/or end of function. But not spread around because then it would become spaghetti